py-utils/src/utils/msg_bus/kafka_python_example.py

```diff
@@ -0,0 +1,55 @@
+from __future__ import absolute_import
+
+from message_bus.bus.bus_frame import Bus
```

Comment:
Please add copyright notice to all the files. Refer to the other cortx-utils files for the format. Copy/Paste from there.

Reply:
copyright added